projects/notebooks/roles/notebooks_dashboard_scale_test/tasks/main.yml

```diff
@@ -184,35 +207,25 @@
 # ---
 
 - name: Create the RHODS test job
-  shell:
-    set -o pipefail;
-    cat "{{ artifact_extra_logs_dir }}/src/000_notebooks_dashboard_scale_test.yaml"
-      | sed 's/$JOB_CREATION_TIME/'$(date "+%Y-%m-%dT%H:%M:%SZ" --utc)'/'
-      | oc create -f- -n "{{ tester_namespace }}"
+  shell: set -o pipefail; cat "{{ artifact_extra_logs_dir }}/src/000_notebooks_dashboard_scale_test.yaml" | sed 's/$JOB_CREATION_TIME/'$(date "+%Y-%m-%dT%H:%M:%SZ" --utc)'/' | oc create -f- -n "{{ tester_namespace }}"
```

Comment:
Here and below, you need to revert the formatting changes, must have been done automatically by your editor :/